.gitignore

```
@@ -1,3 +1,3 @@
 .env
 .DS_Store
-node_modules/
+node_modules
```

```suggestion
node_modules/
```

i think it should include the slash re: https://stackoverflow.com/questions/29820791/git-ignore-node-modules-folder-everywhere and https://github.com/github/gitignore/blob/master/Node.gitignore#L42, is that right or were you experiencing something different?

to overcommunicate, this change doesn't seem relevant to the PR title. isn't mentioned in the commit message. would be easier to review as a separate commit or PR, if is an intended change. definitely no big deal but to communicate what's going through my mind when reviewing the PR